### PR TITLE
mrc-1787 Endpoints for model calibration

### PR DESCRIPTION
**Pull request by EmmaLRussell**

### CODE REVIEW

**Comment by EmmaLRussell** [comment]
I think this is failing because the targeted hintr branch does not support the new coarse-outputs endpoint

File: src/app/src/main/kotlin/org/imperial/mrc/hint/clients/HintrAPIClient.kt [hunk 1]
```diff
@@ -111,6 +119,11 @@ class HintrFuelAPIClient(
         return postJson("model/options", json)
     }
 
+    override fun getModelCalibrationOptions(): ResponseEntity<String>
+    {
+        return postEmpty("model/calibration-options")
```
Comment (hillalex) [inline src/app/src/main/kotlin/org/imperial/mrc/hint/clients/HintrAPIClient.kt:124]: why is this a POST in hintr? seems weird @r-ash ?

**Review by hillalex — APPROVED** [review]
lgtm - but why is fetching calibration options a POST in hintr?

**Comment by hillalex** [comment]
Going to merge this in light of time pressure, but I think we should look into why this endpoint is a POST!

**Pull request merged**